config/routes.rb

```diff
@@ -8,6 +8,10 @@
     mount Sidekiq::Web => '/sidekiq'
   end
 
+  if Rails.env.development?
```

Comment:
Was hoping Heroku review apps were set to dev, I can see now they're on staging :sweat_smile: 
Will need to check the lookbook interface locally

Reply:
```rb
# config/application.rb
config.active_record.verify_foreign_keys_for_fixtures = false
```
Another option to enable in staging I stumbled on if desired